eng/WpfArcadeSdk/tools/SystemResources.props

```diff
@@ -26,7 +26,7 @@
       <ClassName Condition="'$(AssemblyName)'=='ReachFramework'">System.Windows.Xps.SRID</ClassName>
       <ClassName Condition="'$(AssemblyName)'=='PresentationFramework'">System.Windows.SRID</ClassName>
       <ClassName Condition="'$(AssemblyName)'=='PresentationUI'">System.Windows.TrustUI.SRID</ClassName>
-      <ClassName Condition="'$(AssemblyName)'=='WindowsFormsIntegration'">System.Windows.SRID</ClassName>
+      <ClassName Condition="'$(AssemblyName)'=='WindowsFormsIntegration'">System.Windows.SR</ClassName>
```

Comment:
I replaced `SRID` with `SR` because the suffix `ID` does not apply anymore. This is the same as the default in dotnet/runtime: https://github.com/dotnet/runtime/blob/main/eng/resources.targets#L8.